core/platform/auth/firebase_auth_services_test.py

```diff
@@ -519,13 +581,16 @@ def assert_is_not_user_multi(self, uids):
             uids: list(str). The IDs of the users to confirm.
         """
         found = [uid for uid in uids if uid in self._users_by_uid]
+        assert self._test is not None
         self._test.assertEqual(
             found, [],
             msg='Unexpected Firebase accounts exists: uids=%r' % (found,))
 
     def mock_delete_users_error(
             self,
-            batch_error_pattern=(None,), individual_error_pattern=(None,)):
+            batch_error_pattern: Tuple[Optional[Exception]] = (None,),
+            individual_error_pattern: Tuple[Optional[bool]] = (None,)
+    ) -> 'contextlib._GeneratorContextManager[None]':
```

Comment:
```
"ContextManager[List[str]]" of "capture_logging" incompatible with return type "_GeneratorContextManager[Any]" in supertype "TestBase"  [override]

```
If I use `ContextManager[List[str]]` for `capture_logging`. 
Reason: See the type stubs of [contextlib.contextmanager](https://github.com/python/typeshed/blob/ee487304d76c671aa353a15e34bc1102bffa2362/stdlib/contextlib.pyi#L37)

Reply:
Wait, how is `capture_logging` related to this?

Reply:
This error is thrown if I use ContentManager[List[str]] for capture_logging. This means that the self.swap(..) returns `_GeneratorContextManager[...]`. So that's why I used them here?
Should I change everything except capture_logging to use typing.ContextManager?

Reply:
Changed all of these to `ContextManager` except capture_logging. Thanks!